usaspending_api/download/tests/unit/test_file_generation.py

```diff
@@ -5,18 +5,18 @@
 from usaspending_api.download.lookups import VALUE_MAPPINGS
 
 
-def test_get_awards_csv_sources(db):
-    original = VALUE_MAPPINGS["awards"]["filter_function"]
-    VALUE_MAPPINGS["awards"]["filter_function"] = MagicMock(returned_value="")
+def test_get_elasticsearch_awards_csv_sources(db):
+    original = VALUE_MAPPINGS["elasticsearch_awards"]["filter_function"]
+    VALUE_MAPPINGS["elasticsearch_awards"]["filter_function"] = MagicMock(returned_value="")
     csv_sources = download_generation.get_download_sources(
-        {"download_types": ["awards"], "filters": {"award_type_codes": list(award_type_mapping.keys())}}
+        {"download_types": ["elasticsearch_awards"], "filters": {"award_type_codes": list(award_type_mapping.keys())}}
     )
     assert len(csv_sources) == 2
-    VALUE_MAPPINGS["awards"]["filter_function"] = original
+    VALUE_MAPPINGS["elasticsearch_awards"]["filter_function"] = original
```

Comment:
Do you know why these need to be set back to the original value? For both this method and the one below.

Reply:
I removed the `awards` path because it was no longer in use. And when doing so I realized we didn't have a test for the `elasticsearch_transactions` path.